### PR TITLE
doc improvement: Moved the nRF7002EB with nRF54 guides

### PR DESCRIPTION
**Pull request by divipillai**
Integrated the following guides into the nRF7002 EB dev guide:

* nrf7002eb_54h20dk_gs
* nrf7002eb_54l15pdk_gs

### CODE REVIEW

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -218,8 +218,8 @@ nRF IEEE 802.15.4 radio driver
 Wi-Fi
 -----
 
-* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board using :ref:`Shield <ug_nrf7002eb_nrf54h20dk_gs>`.
-* Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board using :ref:`Shield <ug_nrf7002eb_nrf54l15pdk_gs>`.
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`.
+* Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`.
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:222]: @b-gent : Can we keep the change log entry in this PR because removing the PR would fail the doc build.

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 2]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+You can also use the nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15`.
+
 .. figure:: images/nRF7002eb.png
    :alt: nRF7002 EB
 
    nRF7002 EB
 
+PCB edge connector and castellated edge holes
+*********************************************
+
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:22]:
```suggestion
```

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 3]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
+For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
+
 In case of the Nordic Thingy:53, the PCB edge connector connects nRF7002 to nRF5340, which acts as a host.
 For the pinout of the PCB edge connector, see `nRF7002 EB PCB edge connector`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
 For information about how to connect to the Nordic Thingy:53, see `Using nRF7002 EB with the Nordic Thingy:53`_, also in the hardware guide.
 
-The castellated holes on the side of the board allow the EB to be used as a breakout board that can be soldered to other PCB assemblies.
-This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
-For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:26]: Please revert this change, so that the order mentioned in line 23 is kept in the order of paragraphs (edge connector first, castellated holes second).

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 4]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+
+For nRF54H20 DK and for nRF54L15 PDK following are the pin mapping:
+
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:35]:
```suggestion

Pin mapping for the nRF54H20 DK and the nRF54L15 PDK
****************************************************

Given that the hardware documentation for the the nRF54H20 DK and the nRF54L15 PDK is not yet publicly available, refer to the following tables for the pin mapping for these kits:
```

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 5]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+.. tabs::
+
+   .. group-tab:: nRF54H20 DK
+
+       The pin mapping for the nRF54H20 DK is as follows:
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:40]:
```suggestion
```
(This repeats the intro line)

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 6]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+		| IOVDDEN (Not used)                | P1.00             | Enable IOVDD regulator                        |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| IRQ (IRQ)                         | P1.02             | Interrupt from nRF7002                        |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| GRANT (GRT)                       | P1.03             | Coexistence grant from nRF7002                |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| REQ (REQ)                         | P1.08             | Coexistence request to nRF7002                |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| STATUS (ST0)                      | P1.07             | Coexistence status from nRF7002               |
+		+-----------------------------------+-------------------+-----------------------------------------------+
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:64]: I think we don't need the indent here and we can align the table edge to `group-tab`?

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 7]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+
+       .. note::
+	      Connect ``VIO`` to 1.8 V and ``VBAT`` to 3.6 V and ``GND``.
+
+   .. group-tab:: nRF54L15 PDK
+
+       The pin mapping for the nRF54L15 PDK is as follows:
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:71]:
```suggestion
```

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 8]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+		| IOVDDEN (Not used)                | P1.13             | Enable IOVDD regulator                        |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| IRQ (IRQ)                         | P1.14             | Interrupt from nRF7002                        |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| GRANT (GRT)                       | P1.12             | Coexistence grant from nRF7002                |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| REQ (REQ)                         | P1.06             | Coexistence request to nRF7002                |
+		+-----------------------------------+-------------------+-----------------------------------------------+
+		| STATUS (ST0)                      | P1.07             | Coexistence status from nRF7002               |
+		+-----------------------------------+-------------------+-----------------------------------------------+
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:95]: Same note about the table edge indent.

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 9]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
...
+
+	   .. note::
+	      Connect ``VIO`` to 1.8 V and ``VBAT`` to 3.6 V and ``GND``.
 
 .. _nrf7002eb_building_programming:
```
Comment (greg-fer) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:101]: For line 105:
```
To build for the nRF7002 EB, build for the compatible :ref:`board target <app_boards_names>` with the CMake ``SHIELD`` option set to the corresponding shield name.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (wiba-nordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:222]: You can get around the build breaking without the changelog edits by adding these target tags to the `nrf7002eb_dev_guide.rst` if keeping these changes isn't an option.

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst (hunk 6 above)
Comment (wiba-nordic) [reply doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:64]: Table indent seems to be tabs instead of spaces as well, which isn't what we generally use.

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 10]
```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
 The nRF7002 :term:`Expansion Board (EB)` (PCA63561), part of the `nRF70 Series Family <nRF70 Series product page_>`_, can be used to provide Wi-Fi® connectivity to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
 For example, you can use it with the :ref:`Nordic Thingy:53 <ug_thingy53>`, an IoT prototyping platform from Nordic Semiconductor.
 
+You can also use the nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15`.
```
Comment (richabp) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:13]:
```suggestion
You can also use the nRF7002 EB to provide Wi-Fi connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15`.
```

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst (hunk 4 above)
Comment (richabp) [reply doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:35]: Is it important for the user to know that the hardware docs are not available yet?

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 11]
```diff
@@ -46,3 +118,20 @@ Alternatively, add the shield in the project's :file:`CMakeLists.txt` file by us
...
+Alternatively, you can add the shield in the project's :file:`CMakeLists.txt` file by using the ``set(SHIELD nrf700x_nrf54l15pdk)`` command.
+
+Limitations when building with nRF54H20 DK and nRF54L15 PDK
+***********************************************************
+
+The Wi-Fi support is experimental and has the following limitations:
+
+* It only supports STA mode.
+* It is only suitable for low-throughput applications.
+* For nRF54L15 PDK, it only allows Wi-Fi open security
```
Comment (richabp) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:137]:
```suggestion
* For nRF54L15 PDK, it only allows Wi-Fi open security.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (richabp) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:222]: I think we can combine the entries to avoid repetition.

"* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` and:ref:`zephyr:nrf54l15pdk_nrf54l15` boards with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`."

**Review by richabp — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (divipillai) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: > You can get around the build breaking without the changelog edits by adding these target tags to the `nrf7002eb_dev_guide.rst` if keeping these changes isn't an option.

Yeah, I had thought of this. But, since these pages were recently added, thought of getting rid of these labels. But since you also think we could go with this option, I have added these labels to the nrf7002eb_dev_guide.rst. This also avoids editing some of the other pages as well.

Comment (divipillai) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: > I think we can combine the entries to avoid repetition.
> 
> "* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15` boards with :ref:`nRF7002 EB <ug_nrf7002eb_gs>`."

I will add a suggestion to PR https://github.com/nrfconnect/sdk-nrf/pull/15961, since this is now removed from this PR.

**Review by wiba-nordic — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by richabp — APPROVED** [review]
Thanks, Divya.

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 12]
```diff
@@ -46,3 +115,20 @@ Alternatively, add the shield in the project's :file:`CMakeLists.txt` file by us
...
+Alternatively, you can add the shield in the project's :file:`CMakeLists.txt` file by using the ``set(SHIELD nrf700x_nrf54l15pdk)`` command.
+
+Limitations when building with nRF54H20 DK and nRF54L15 PDK
+***********************************************************
+
+The Wi-Fi support is experimental and has the following limitations:
+
+* It only supports STA mode.
+* It is only suitable for low-throughput applications.
+* For nRF54L15 PDK, it only allows Wi-Fi open security.
```
Comment (krish2718) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:134]: This is now updated.

`* For nRF54L15 PDK, WPA3 security isn't supported`

File: doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst [hunk 13]
```diff
@@ -46,3 +115,21 @@ Alternatively, add the shield in the project's :file:`CMakeLists.txt` file by us
...
```
Comment (krish2718) [inline doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst:134]: This statement is'nt needed now.
